race.py

```diff
@@ -0,0 +1,60 @@
+import asyncio
+import operator
+import time
+
+from constants import TIME_FORMAT, ORIGIN0_OFFSET
+from prerace_setup import initialize_rounds, load_race_settings, unpack_round_settings
+
+RACER_RUNTIME = 0
+RACER_NAME = 1
+
+
+def race():
+    race_settings = load_race_settings()
+    rounds = initialize_rounds(race_settings)
+
+    loop = asyncio.get_event_loop()
+    for round_num in rounds:
+        curr_round = rounds[round_num]
```

Comment:
is there a reason rounds is a dict and not a list (honestly, even a generator?)
iterating would be easier that wat